llvm/lib/Analysis/TargetTransformInfo.cpp

```diff
@@ -17,7 +17,6 @@
 #include "llvm/IR/IntrinsicInst.h"
 #include "llvm/IR/Module.h"
 #include "llvm/IR/Operator.h"
-#include "llvm/IR/PatternMatch.h"
```

Comment:
looks like accidental change

Reply:
This header is not used directly, so I thought of removing it.
It's unrelated to my patch though.